src/frontend/komponenter/Fagsak/Vilkaar/VelgVilkaarene.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { IFagsak, IBehandling, IVedtakForBehandling, VedtakResultat } from '../../../typer/fagsak';
+import { useHistory } from 'react-router';
+import { Knapp } from 'nav-frontend-knapper';
+import { Systemtittel, Normaltekst } from 'nav-frontend-typografi';
+import { RadioPanelGruppe } from 'nav-frontend-skjema';
+import { axiosRequest } from '../../../api/axios';
+import { Ressurs, RessursStatus } from '../../../typer/ressurs';
+import AlertStripe from 'nav-frontend-alertstriper';
+
+interface IProps {
+    fagsak: IFagsak;
+}
+
+const VelgVilkaarene: React.FunctionComponent<IProps> = ({ fagsak }) => {
+    const history = useHistory();
+    const [fyltvilkår, settFyltvilkår] = React.useState(true);
+    const [feilmelding, settFeilmelding] = React.useState('');
+    const [senderInn, settSenderInn] = React.useState(false);
+
+    const aktivBehandling = fagsak.behandlinger.find(
+        (behandling: IBehandling) => behandling.aktiv === true
+    );
+
+    const aktivVedtak = aktivBehandling
+        ? aktivBehandling.vedtakForBehandling.find(
+              (vedtak: IVedtakForBehandling) => vedtak.aktiv === true
+          )
+        : undefined;
+
+    React.useEffect(() => {
+        if (aktivVedtak) {
+            settFyltvilkår(aktivVedtak.resultat == VedtakResultat.INNVILGET);
+        }
+    }, []);
+
+    if (!aktivBehandling) {
+        return (
+            <div>
+                <Normaltekst>Ingen aktiv behandling</Normaltekst>
+            </div>
+        );
+    }
+
+    return (
+        <div>
+            {feilmelding !== '' && <AlertStripe type="feil">{feilmelding}</AlertStripe>}
+            <div className={'vilkårene'}>
+                <Systemtittel children={'Inngangsvilkår'} />
+
+                <br />
+                <RadioPanelGruppe
+                    name={'inngangsvilkår'}
+                    legend={'Inngangsvikår for barnetrygd'}
+                    radios={[
+                        { label: 'Inngangsvilkårene er oppfylt', value: 'oppfylt' },
```

Comment:
I think we should create an enum for this and use that value for the radios. This way we have the possibility to add or remove values to the enum without changing this code 👍

Reply:
agree👍